src/utils/bookmakers.ts

```diff
@@ -40,134 +57,97 @@ export const checkOddsFromBookmakers = (
         return false;
     });
 
-    if (arrayOfBookmakers.length == 1) {
-        if (hasZeroOrOne) {
-            // If any bookmaker has zero odds, return zero odds
-            return [
-                {
-                    homeOdds: 0,
-                    awayOdds: 0,
-                    drawOdds: 0,
-                },
-            ];
-        }
+    if (hasZeroOrOne) {
+        // If any bookmaker has zero odds, return zero odds
+        return {
+            homeOdds: 0,
+            awayOdds: 0,
+            drawOdds: 0,
+            errorMessage: 'Returning zero odds cause bookmakers have 0 or 1 odds',
```

Comment:
Add bookmaker in error message.

Reply:
Added TODO to add this after release with first refactoring